app/controllers/api/v1/profiles_controller.rb

```diff
@@ -51,21 +51,11 @@ def create
       private
 
       def set_profile
-        @profile = Profile.find(params[:id])
-      end
-
-      def serialize_profile
-        ProfileSerializer.new(@profile).to_json
-      end
-
-      def serialize_profiles
-        @profiles.map { |profile| ProfileSerializer.new(profile) }.to_json
+        @profile = Profile.find_by(user_id: params[:user_id]) || Profile.find(params[:id])
```

Comment:
we can get profile from  a user.
as soon as it's sunder resources user scope, we should define URLs like `users/:id/profile` and make just
```
Profile.find_by(user_id: ...)
```